string path
uint32 numpoints_thresh
uint32 outlook_thresh
float32 deviation_thresh

